### DISCUSSION TITLE
primary 和parallel 中 ”_return“ 的作用范围。

### DISCUSSION THREAD

**Discussion by wy16W2pIilK1xgqN** [IrineSistiana/mosdns#478] [Q&A] [answered]
文档中提到在 sequence 使用 ”_return“，会跳出整个  sequence 。
而在 primary 和parallel 的介绍中 说是独立序列，使用  ”_return“ 只会跳出 当前序列
而 primary 和 paralle 作为  sequence  的子项目，在其中使用  ”_return“  否会跳出  sequence  ，还是只跳出   primary 和 paralle ？

**Comment by IrineSistiana** [comment, marked as answer]
只跳出 primary 和 paralle 。

wiki 这部分更新了一些。
